akka-stream/src/main/boilerplate/akka/stream/FanInShape.scala.template

```diff
@@ -7,18 +7,24 @@ import scala.collection.immutable
 import scala.annotation.varargs
 
 object FanInShape {
-  sealed trait Init[+O]
-  case class Name(name: String) extends Init[Nothing]
-  case class Ports[O](out: Outlet[O], ins: immutable.Seq[Inlet[_]]) extends Init[O]
+  sealed trait Init[+O] {
```

Comment:
Not the Tuple3?

-- 
Cheers,
√
On 29 Jun 2015 15:56, "Roland Kuhn" <notifications@github.com> wrote:

> In akka-stream/src/main/boilerplate/akka/stream/FanInShape.scala.template
> <https://github.com/akka/akka/pull/17856#discussion_r33465676>:
>
> > @@ -7,18 +7,24 @@ import scala.collection.immutable
> >  import scala.annotation.varargs
> >
> >  object FanInShape {
> > -  sealed trait Init[+O]
> > -  case class Name(name: String) extends Init[Nothing]
> > -  case class Ports[O](out: Outlet[O], ins: immutable.Seq[Inlet[_]]) extends Init[O]
> > +  sealed trait Init[+O] {
>
> The number of objects created and retained is the same as before, but I
> don't mind either way.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/akka/akka/pull/17856/files#r33465676>.
>